extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -15,8 +15,11 @@ class RecirculationHooks {
 	 * @return bool
 	 */
 	public static function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
+		global $wgDevelEnvironment;
+
 		JSMessages::enqueuePackage( 'Recirculation', JSMessages::EXTERNAL );
 		Wikia::addAssetsToOutput( 'recirculation_scss' );
+		$out->addJsConfigVars('wgServiceUrl', empty( $wgDevelEnvironment ) ? 'services.wikia.com' : 'services.wikia-dev.pl' );
```

Comment:
`wgServicesExternalDomain` is already available on the client side and should be used instead